lib/error.js

```diff
@@ -41,6 +41,12 @@ var ERRNO = {
   BOUNCE_SOFT: 135,
   SERVER_BUSY: 201,
   FEATURE_NOT_ENABLED: 202,
+  EMAIL_EXISTS: 136,
```

Comment:
The obsessive-compulsive in me wants these to be inserted in their correct position, between `BOUNCE_SOFT` and `SERVER_BUSY`.